app/models/miq_product_feature/seeding.rb

```diff
@@ -0,0 +1,30 @@
+class MiqProductFeature < ApplicationRecord
+  module Seeding
+    extend ActiveSupport::Concern
+
+    RELATIVE_FIXTURE_PATH = "db/fixtures/miq_product_features".freeze
+    FIXTURE_PATH = Rails.root.join(RELATIVE_FIXTURE_PATH).freeze
+
+    module ClassMethods
+      def seed_files
+        return seed_root_filename, seed_core_files + seed_plugin_files
+      end
+
+      private
+
+      def seed_root_filename
+        "#{FIXTURE_PATH}.yml"
+      end
+
+      def seed_core_files
+        Dir.glob("#{FIXTURE_PATH}/*.y{,a}ml").sort
```

Comment:
I appreciate the sort on Dir.glob  ❤️

Reply:
I basically *always* sort that method now, because of its unpredictability... I'm almost tempted to create a more_core_extensions method that enforces it :)